docs/content.zh/docs/dev/datastream/execution_mode.md

```diff
@@ -201,11 +201,11 @@ Checkpointing 用于故障恢复的特点之一是，在发生故障时，Flink
 
 `批`模式下的行为变化：
 
-* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 [sum()]({{< ref "docs/dev/datastream/operators/overview" >}}#aggregations)，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
+* “滚动"操作，如 [reduce()]({{< ref "docs/dev/datastream/operators/overview" >}}#reduce) 或 sum()，会对`流`模式下每一条新记录发出增量更新。在`批`模式下，这些操作不是"滚动”。它们只发出最终结果。
```

Comment:
Why remove the `sum()` link?

Reply:
Glad to get your review .  @wuchong 

There isn't  any aggregate function section title in the docs/dev/datastream/operators/overview page, but we could make it mapping to the page   [flink/docs/content.zh/docs/dev/datastream/operators/windows.md](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/operators/windows/#aggregatefunction') .  

Please let me know what's your opinion.

Reply:
The English documentation doesn't have a link to `sum()` so I think it's fine to remove the link here